cpu/stm32/include/vendor/system_stm32mp1xx.h

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
You don't need this file.

Reply:
It is needed as it is included in `cpu/stm32/include/vendor/stm32mp157cxx_cm4.h`

Reply:
Ah ok, then I'm fine with removing the include in `stm32mp157cxx_cm4.h`.